--- gnuplot-4.0.0/Makefile.am	2012-03-08 10:35:48.467421000 +0100
+++ gnuplot-4.0.0/Makefile.am	2004-02-19 23:42:44.000000000 +0100
@@ -1,7 +1,7 @@
 ## Process this file with automake to produce Makefile.in -*-Makefile-*-
 AUTOMAKE_OPTIONS = foreign 1.2h

-SUBDIRS = config m4 term src docs lisp man demo tutorial
+SUBDIRS = config m4 term src

 EXTRA_DIST = BUGS CodeStyle Copyright FAQ GNUmakefile INSTALL INSTALL.gnu \
 Makefile.maint PATCHLEVEL PGPKEYS PORTING README README.1ST README.exp \
